server/entities/status.lua

```diff
@@ -0,0 +1,25 @@
+-- Iport section
+
+Colors = require("graphics.colors")
+
+--
+
+local states = {
+    ON = {name = "ON", color = Colors.workingColor},
+    IDLE = {name = "IDLE", color = Colors.idleColor},
+    OFF = {name = "OFF", color = Colors.offColor},
+    BROKEN = {name = "BROKEN", color = Colors.errorColor},
+    MISSING = {name = "NOT FOUND", color = Colors.errorColor}
+}
```

Comment:
Colors here are probably not needed and the front end could parse the states into colors later